### DISCUSSION TITLE
Is there a guide/wiki mention on to use MongoDB on KeystoneJS 6?

### DISCUSSION THREAD

**Discussion by AshKetshup** [keystonejs/keystone#9374] [Questions] [answered]
I've been trying to migrate a KeystoneJS 5 & MongoDB 4.2 project into KeystoneJS 6 & MongoDB 6.0 but I'm having a bunch of problems. Is there a guide to use MongoDB or is it not possible? I've been looking into the docs but haven't found much on it.

**Comment by marekryb** [comment, marked as answer]
MongoDB is not supported in Keystone 6.

See https://github.com/keystonejs/keystone/discussions/7511#discussioncomment-3430541
